videoframes.py

```diff
@@ -45,6 +50,10 @@ def setup_ui(self):
         else:
             sys.exit("Platform unsupported.")
 
+# sets the voulme with the value of the slider(0-100)
```

Comment:
Add this comment as a docstring of the method instead :)

So instead of: 
```python
# sets the voulme with the value of the slider(0-100)
def set_volume(self):
    self.player.audio_set_volume(self.volume_slider.value())
``` 
it would become 
```python
def set_volume(self):
    """Sets the volume according to the range of the UI volume slider."""
    self.player.audio_set_volume(self.volume_slider.value())
``` 
and be placed inside the method instead. See the setup method for an example. :)